doc/maintaining/cli.rst

```diff
@@ -0,0 +1,558 @@
+.. _cli:
+
+======================
+Command Line Interface (CLI)
+======================
+
+NB: From CKAN 2.9 onwards the CKAN configuration file is named 'ckan.ini'. Previous names: 'production.ini' and 'development.ini' (plus others) may also still appear in documentation and the software. These legacy names will eventually be phased out. Also from CKAN 2.9 onwards, the `paster`` command used for common CKAN administration tasks has been replaced with the  ``ckan`` command.
```

Comment:
It would be more consistent to use `.. note::` Also:
```suggestion
NB: From CKAN 2.9 onwards the CKAN configuration file is named 'ckan.ini'. Previous names: 'production.ini' and 'development.ini' (plus others) may also still appear in documentation and the software. These legacy names will eventually be phased out. Also from CKAN 2.9 onwards, the ``paster`` command used for common CKAN administration tasks has been replaced with the  ``ckan`` command.
```